### PR TITLE
Edits for neutral point of view in truthy/falsy

### PR DESCRIPTION
**Pull request by jduckles**
Per #29 updating truthy/falsy definitions to be neutral in their point of view. 

## Author: 

- Jonah Duckles

## Language: 

- English 

## Terms defined:

None

## Editor

Assign the PR based on the language to the following person:

| Language | Person | 
|:--------:|:------:|
| English  | @zkamvar |
| Spanish  | @ian-flores |
| French   | @fmichonneau |

### CODE REVIEW

**Comment by zkamvar** [comment]
I know I cannot speak for everyone, but I was unaware of what "Boolean" meant until I was in grad school. What do you think about simplifying the definitions (truthy: "evaluates to true" or "colloquially thought to be true" and/or define boolean as well.

**Comment by jduckles** [comment]
@zkamvar Updated to add a Boolean definition and associated references.

File: glossary.yml [hunk 1]
```diff
@@ -592,9 +602,10 @@
   en:
     term: "falsy"
     def: >
-      A horrible neologism meaning "equivalent to false".
+      Evaluating to false in a Boolean context.
```
Comment (zkamvar) [inline glossary.yml:605]:
```suggestion
      Evaluating to false in a [Boolean](#boolean) context. 
```

File: glossary.yml [hunk 2]
```diff
@@ -2086,10 +2097,11 @@
   en:
     term: "truthy"
     def: >
-      A truly Orwellian neologism meaning "not equivalent to false". Cf.
-      [falsy](#falsy), but only if you are able to set aside your respect for the
-      English language.
-
+      Evaluating to true in a Boolean context.
```
Comment (zkamvar) [inline glossary.yml:2100]:
```suggestion
      Evaluating to true in a [Boolean](#boolean) context.
```

**Comment by zkamvar** [comment]
Thank you! I have one more suggestion for cross-linking, purposes, but that should be it! (you can safely ignore the lint error, that should be fixed in another PR).

**Comment by jduckles** [comment]
@zkamvar I've added a few more deffinitions to build-out the thread of boolean logic. Still more to add in the future though, bytes, kilobytes, megabytes... etc.

File: glossary.yml [hunk 3]
```diff
@@ -183,12 +183,43 @@
   ref:
     - tidyverse
 
+- slug: binary
+  en:
+    term: binary
+    def: > 
+      A system which can have one of two possible states. In computing often represented as being in the state 0 or 1.
+      Represented in Boolean logic as False (0) or True (1). Computers are built upon systems which store 0s and 1s as "bits"
```
Comment (zkamvar) [inline glossary.yml:191]:
```suggestion
      Represented in [Boolean](#boolean) logic as [False](#false) (0) or [True](#true) (1). Computers are built upon systems which store 0s and 1s as "[bits](#bit)"
```

File: glossary.yml [hunk 4]
```diff
@@ -183,12 +183,43 @@
...
+   ref:
+     - boolean
+     - bit
+
+- slug: bit
+  en:
+    term: bit
+    def: > 
+      A unit of information representing representing alternatives, yes/no, true/false. In computing a state of either 0 or 1.
```
Comment (zkamvar) [inline glossary.yml:200]:
```suggestion
      A unit of information representing representing alternatives, yes/no, [true](#true)/[false](#false). In computing a state of either 0 or 1. 
```

File: glossary.yml [hunk 5]
```diff
@@ -183,12 +183,43 @@
...
   en:
     term: "boilerplate"
     def: >
       FIXME
 
+- slug: boolean
+  en:
+    term: "Boolean"
+    def: > 
+       Relating to a variable or data type that can have either a logical value of true or a value of false. Named for George Boole,
```
Comment (zkamvar) [inline glossary.yml:215]:
```suggestion
       Relating to a variable or data type that can have either a logical value of [true](#true) or a value of [false](#false). Named for George Boole, 
```

File: glossary.yml [hunk 6]
```diff
@@ -588,13 +619,26 @@
     def: >
       FIXME
 
+- slug: "false"
+  en: 
+    term: "false"
+    def: > 
+       The logical (Boolean) state opposite of "true". Used in logic and programming to represent binary state of something.
```
Comment (zkamvar) [inline glossary.yml:626]:
```suggestion
       The logical ([Boolean](#boolean)) state opposite of "[true](#true)". Used in logic and programming to represent [binary](#binary) state of something. 
```

File: glossary.yml [hunk 7]
```diff
@@ -588,13 +619,26 @@
...
+     - true
+     - truthy
+     - falsy 
+
 - slug: falsy
   en:
     term: "falsy"
     def: >
-      A horrible neologism meaning "equivalent to false".
+      Evaluating to a state of false in a Boolean context.
```
Comment (zkamvar) [inline glossary.yml:638]:
```suggestion
      Evaluating to [false](#false) in a [Boolean](#boolean) context. 
```

Making this consistent with the true definition

File: glossary.yml [hunk 8]
```diff
@@ -2082,14 +2126,28 @@
     def: >
       FIXME
 
+- slug: "true" 
+  en: 
+  term: "true" 
+  def: >
+     The logical (Boolean) state opposite of "false". Used in logic and programming to represent binary state of something.
```
Comment (zkamvar) [inline glossary.yml:2133]:
```suggestion
     The logical ([Boolean](#boolean)) state opposite of "[false](#false)". Used in logic and programming to represent [binary](#binary) state of something. 
```

File: glossary.yml [hunk 9]
```diff
@@ -588,13 +619,26 @@
     def: >
       FIXME
 
+- slug: "false"
+  en: 
+    term: "false"
+    def: > 
+       The logical (Boolean) state opposite of "true". Used in logic and programming to represent binary state of something. 
+       Often fluidly represented as true/false, on/off, 1/0.
```
Comment (zkamvar) [inline glossary.yml:627]:
```suggestion
```

False is rarely represented as "true" unless it's coming from a politician or [a malicious R package](https://github.com/Ironholds/jammr)

File: glossary.yml [hunk 10]
```diff
@@ -2082,14 +2126,28 @@
     def: >
       FIXME
 
+- slug: "true" 
+  en: 
+  term: "true" 
+  def: >
+     The logical (Boolean) state opposite of "false". Used in logic and programming to represent binary state of something. 
+      Often fluidly represented as true/false, on/off, 1/0.
```
Comment (zkamvar) [inline glossary.yml:2134]:
```suggestion
```

**Comment by zkamvar** [comment]
How fun! I approve the merge, yet I'm not a maintainer :sweat_smile: @gvwilson, @fmichonneau @ian-flores, do any of you have access to merge this?

**Comment by gvwilson** [comment]
I think @zkamvar should be a maintainer - @fmichonneau ?

**Comment by fmichonneau** [comment]
thank you all!
there were a few issues with the formatting of the yaml file that I fixed.
@zkamvar (as well as @gvwilson and @ian-flores) are now "admin" on all 3 glosario-related repos.

**Pull request merged**